packages/editor/src/client/panel/imageUploadAndEditPanel.tsx

```diff
@@ -2,21 +2,90 @@ import React, {useState} from 'react'
 
 import {ImagedEditPanel} from './imageEditPanel'
 import {ImageUploadPanel} from './imageUploadPanel'
+import exifr from 'exifr'
 
 export interface ImageUploadAndEditPanelProps {
   onClose(): void
   onUpload(): void
 }
 
+export type ImageMetaData = {
+  [key: string]: string
+}
+
+type metaTagsMap = {
+  [key: string]: string[]
+}
+
+const DEFAULT_META_TAG_MAP: metaTagsMap = {
```

Comment:
idem

Reply:
idem

Reply:
@michael-scheurer as discusted over the phone :-)